### ISSUE TITLE
ASC evidence, mismatch between disease and mapped EFO?

### ISSUE THREAD

**Issue by ravwojdyla** [opentargets/issues#2687]
I'm using 22.06 release data, and looking at the Gene Burden data source. It seems like for all lines of evidence coming from ASC (Autism Sequencing Consortium) the `diseaseFromSource` is `autism spectrum disorder`, but EFO is mapped to very different phenotypes.

**Comment by ireneisdoomed** [comment]
Hi @ravwojdyla,

this is indeed a bug in our manual annotation. The only valid EFO term is [EFO_0003756](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0003756). This incorrect annotation is affecting 66 out of 67 ASC evidence.

The data has been fixed and the updated evidence will become available in our next release. We have also implemented a data sanity check so that this won't happen again.

Apologies for the inconvenience and thank you so much for the report!
